ocfweb/docs/docs/services/account/content-removal.md

```diff
@@ -0,0 +1,54 @@
+[[!meta title="Content Removal Policy"]]
+
+
+## Introduction
+
+Most of the content on [ocf.berkeley.edu](https://ocf.berkeley.edu)
```

Comment:
can you make this a link to www.ocf.berkeley.edu? it's fine if the text still says ocf.b.e.